### PR TITLE
config: test +HHMM offset in ReadableSchedule

### LINKED ISSUE #17080: config: Minor: test +HHMM in  ReadableSchedule
# config: test both formats +HHMM offsets and as well as +HH:MM for ReadableSchedule

Our documents refer to +HHMM format but out tests only test +HH:MM format for offsets

### PR DESCRIPTION
**Pull request by afeinberg**
### What is changed and how it works?

Issue Number: Close #17080

What's Changed:

```commit-message
* Test both formats +HHMM offsets and as well as +HH:MM for ReadableSchedule.
```

### Related changes

- [ ] PR to update `pingcap/docs`/`pingcap/docs-cn`:
- [ ] Need to cherry-pick to the release branch

### Check List

Tests 

- [X] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

### Release note

```release-note
None
```

### CODE REVIEW

**Review by v01dstar — APPROVED** [review]
lgtm

File: components/tikv_util/src/config.rs [hunk 1]
```diff
@@ -2069,7 +2069,7 @@ mod tests {
     #[test]
     fn test_readable_schedule() {
         let schedule = ReadableSchedule(
-            vec!["09:30 +00:00", "23:00 +00:00"]
```
[the remark this answers is not in the document]
Comment (afeinberg) [reply components/tikv_util/src/config.rs:2072]: Done.

**Review by SpadeA-Tang — APPROVED** [review]

**Review by LykxSassinator — APPROVED** [review]

**Pull request merged**